<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>.AppleHongKongChineseFont</family>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.SF Hebrew</family>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>五子棋</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>277</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_player">
      <property name="font">
       <font>
        <family>Apple LiGothic</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>玩家模式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_ai">
      <property name="font">
       <font>
        <family>Apple LiGothic</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>AI模式（玩家先手）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_ai_2">
      <property name="font">
       <font>
        <family>Apple LiGothic</family>
        <pointsize>28</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>AI模式（玩家后手）</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
